qcommon/src/main/java/jake2/qcommon/network/messages/server/ServerMessage.java

```diff
@@ -10,6 +13,121 @@ public ServerMessage(ServerMessageType type) {
         this.type = type;
     }
 
+    /*
+     * ================= CL_ParseEntityBits
+     *
+     * Returns the entity number and the header bits =================
+     */
+    @Deprecated
```

Comment:
Why is it deprecated now?

Reply:
it's a shared piece used by spawn_baseline & packet_entities messages. It is still used in packet_entities messages (which will be fixed in #47 )